pynxtools/dataconverter/readers/xps/reader.py

```diff
@@ -366,29 +352,30 @@ def read(self,
                 with open(file, mode="r", encoding="utf-8") as eln:
                     eln_data_dict = flatten_and_replace(
                         FlattenSettings(
-                            yaml.safe_load(eln),
-                            CONVERT_DICT,
-                            REPLACE_NESTED
+                            yaml.safe_load(eln), CONVERT_DICT, REPLACE_NESTED
                         )
                     )
-            elif file_ext in [".sle", ".xml", ".txt"]:
-                data_dict = XpsDataFileParser([file]).get_dict(**kwargs)
+            elif file_ext in [".sle", ".xml", ".txt", "xy", "vms"]:
```

Comment:
`__prmt_file_extension__` from `XPSDataFileparser` class., then Afterwards we modify one place for file extension. Or one modulelevel variable and can use everywhere you need.

Reply:
Sounds good, I used `_prmt_file_extension__` from `XPSDataFileparser`.